web-client/src/components/CurrentVersion.jsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import '../App.css';
+import { makeStyles } from '@material-ui/core/styles';
+import Button from '@material-ui/core/Button';
+import Rating from '@material-ui/lab/Rating';
+import Tooltip from '@material-ui/core/Tooltip';
+import Box from '@material-ui/core/Box';
+
+function CurrentVersion(props){
+
+    const styles = makeStyles({
+        currentVersion:
+        {
+            width: '80%',
+            float: 'left',
+            marginTop: '2.5%',
+            marginBottom: '2.5%',
+            textAlign: 'left',
+            marginLeft: '5%',
+        },
+        review:
+        {
+            width: '100%',
+            float: 'left',
+        },
+        leftReview:
+        {
+            width: '15%',
+            float: 'left',
+        },
+        rightReview:
+        {
+            width: '80%',
+            float: 'right',
+        },
+        labelVersion:
+        {
+            paddingLeft: '10%',
+        },
+        image:
+        {
+            width: 80,
+            height: 80,
+            borderRadius: '50%',
+            float: 'center',
+            marginLeft: '20%',
+        },
+        me:
+        {
+            width: '100%',
+            float: 'left',
+            textAlign: 'center',
+        },
+        rightDate:
+        {
+            float: 'right',
+            color: 'grey',
+        },
+        textReview:
+        {
+            width: '95%',
+            float: 'left',
+            paddingLeft: '2%',
+            fontSize: '16px',
+            paddingBottom: '2%',
+        },
+    });
+
+    const style = styles();
+
+    let d = new Date();
+    let hours = `${d.getHours()}:${d.getMinutes()}`;
+    let date = `${d.getDate()}/${d.getMonth()+1}/${d.getFullYear()}`;
+
+    const [hover] = React.useState(1);
```

Comment:
Same as line 134 in CurrentVersionWithReplyToReview.jsx

Reply:
You're right so I deleted it